apps/cowswap-frontend/src/modules/notifications/hooks/useTgAuthorization.ts

```diff
@@ -8,13 +8,38 @@ const TG_SESSION_CHECK_INTERVAL = ms`3s`
 
 const ENV_TG_BOT_ID = process.env.REACT_APP_TG_BOT_ID
 const TG_BOT_ID = ENV_TG_BOT_ID ? parseInt(ENV_TG_BOT_ID) : 7076584722 // cowNotificationsBot
+const TG_DEV_BYPASS = process.env.NODE_ENV === 'development' && process.env.REACT_APP_TG_DEV_BYPASS === 'true'
```

Comment:
This code is duplicated.
Can you move to a central location to avoid repetition and potential bugs?